mne/dipole.py

```diff
@@ -445,11 +470,22 @@ def fit_dipole(evoked, cov, bem, trans=None, n_jobs=1, verbose=None):
     """
     # This could eventually be adapted to work with other inputs, these
     # are what is needed:
+
+    evoked.info['projs'] = []  # get rid of SSP projections
```

Comment:
we should check to see what we do in the inverse code, it seems like we should do the same thing in both places. so if we don't do it there, maybe we should start...?